### PR TITLE
Update readthedocs url to a working one

### PR DESCRIPTION
**Pull request by MVrachev**

### CODE REVIEW

**Review by kairoaraujo — APPROVED** [review]

**Review by KAUTH — CHANGES_REQUESTED** [review]
Isn't the Development Guide URL this one, https://repository-service-tuf.readthedocs.io/en/latest/devel/index.html?

**Comment by kairoaraujo** [comment]
> Isn't the Development Guide URL this one, https://repository-service-tuf.readthedocs.io/en/latest/devel/index.html?

Good point @KAUTH, I didn't notice it.

File: README.rst [hunk 1]
```diff
@@ -82,6 +82,6 @@ Check our `CONTRIBUTING.rst <CONTRIBUTING.rst>`_ for more details on how to
 contribute.
 
 Please, check the `Repository Service for TUF Development Guide
-<https://repository-service-tuf.readthedocs.org/devel>`_.
+<https://repository-service-tuf.readthedocs.io/>`_.
```
Comment (kairoaraujo) [inline README.rst:85]:
```suggestion
<https://repository-service-tuf.readthedocs.io/en/latest/devel/>`_.
```

**Review by kairoaraujo — CHANGES_REQUESTED** [review]

**Comment by MVrachev** [comment]
Okay, so the link I proposed is the official one and the one @KAUTH suggested is the development guide?
What's the difference between the two?

**Comment by KAUTH** [comment]
> Okay, so the link I proposed is the official one and the one @KAUTH suggested is the development guide? What's the difference between the two?

AFAIU, I believe https://repository-service-tuf.readthedocs.io/ gives a general overview of RSTUF and its use-case (more for general users) and the other link is more focused on the development of RSTUF (more for contributors).

**Review by kairoaraujo — APPROVED** [review]

**Comment by MVrachev** [comment]
I am working on a solution for the CI.
When we have it I will rebase the pr.

**Review by KAUTH — DISMISSED** [review]

**Comment by MVrachev** [comment]
I am sorry, but I had to rebase on top of the latest changes in the main branch which include a fix for the CI.
The approval status of your reviews has been dismissed by GitHub, so if you could give it again will be awesome.

**Comment by KAUTH** [comment]
Sure thing @MVrachev!

**Review by KAUTH — APPROVED** [review]

**Review by kairoaraujo — APPROVED** [review]

**Pull request merged**